scripts/o3de/tests/test_project_properties.py

```diff
@@ -37,7 +45,7 @@
 '''
 
 
-@pytest.fixture(scope='class')
+@pytest.fixture(scope='function')
```

Comment:
I assuming we had to change the scope to function because subsequent test was modifying, the project json daa in a way that would cause further test to fail?

According to the pytest documentation, 'function'; is the default scope and therefore we can remove it https://docs.pytest.org/en/6.2.x/fixture.html#fixture-scopes

Reply:
You are correct, persisting data was causing tests to fail that shouldn't